packages/app/src/app/pages/Sandbox/Editor/Workspace/Project/Frozen/index.tsx

```diff
@@ -1,47 +1,68 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { observer } from 'mobx-react-lite';
 import Switch from '@codesandbox/common/lib/components/Switch';
 import Tooltip from '@codesandbox/common/lib/components/Tooltip';
 
-import { useSignals } from 'app/store';
+import { useSignals, useStore } from 'app/store';
 
 import { Item, PropertyValue, PropertyName, Icon } from '../elements';
-import { FreezeContainer } from './elements';
+import { FreezeContainer, FrozenWarning } from './elements';
 
 type Props = {
   frozen: boolean;
 };
 
 const FrozenComponent = ({ frozen }: Props) => {
   const { editor } = useSignals();
+  const {
+    editor: { currentSandbox, sessionFrozen },
```

Comment:
@SaraVieira I think you should destructur `currentSandbox` a bit further

```suggestion
    editor: {
      currentSandbox: { customTemplate },
      sessionFrozen,
    },
```

Reply:
I didn't do it like that because in my and ives idea makes it more confusing